configs/sst_conversions.yaml

```diff
@@ -0,0 +1,30 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: RHC Worker Bash
+  description: Remote Host Configuration (rhc) worker for executing bash scripts on hosts managed by Red Hat Insights.
+  maintainer: sst_conversions
+
+  packages:
+    - rhc-worker-bash
+  
+  labels:     
+  - c9s # TODO(r0x0d): Verify if this is the correct label for RHEL 7
```

Comment:
RHEL 7 doesn't have a CentOS Stream equivalent.  This is the correct label for anything going into CentOS Stream/RHEL 9.

Reply:
Oh right, I just want to deploy this to EL7 for now, CentOS 7 will be needed as well, but not sure if we will need it there right away. @bocekm, would you know?

Reply:
We need the worker package in RHEL 7 only. We're working with the Software Production team to have the package copied over to a new unprotected CDN repository with every new version of the package (https://issues.redhat.com/browse/RHELDST-18108). We don't care if the package lands in CentOS Stream, Rocky, Alma, or other distros.